Makefile

```diff
@@ -200,7 +200,7 @@ operator-sdk: $(OPERATOR_SDK) ## Download operator-sdk locally if necessary.
 
 CONTROLLER_GEN = $(PROJECT_PATH)/bin/controller-gen
 $(CONTROLLER_GEN):
-	$(call go-install-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.14.0)
+	$(call go-install-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.16.5)
```

Comment:
Using latest released version of cert manager bumped k8s dependencies which required bumping controller-gen version to work correctly

Reply:
hmm, maybe this could be done in it's own PR? One that only does this?

Reply:
Yeah, I can break it out if you want but it'll possibly prevent this PR from merging sooner, since it'll need a rebase on top once that's merged 🤔

Reply:
I guess if time is an issue we can go with it as is, but generally these things would be better done in there own PRs